### DISCUSSION TITLE
Unknown package

### DISCUSSION THREAD

**Discussion by Shamaoke** [ocaml/dune#7001] [Q&A] [answered]
Hi.

I'm trying to install the [base](https://opam.ocaml.org/packages/base) package using the `dune install base` command. However, the command fails with the error: `Error: Unknown package base!`.

Why this happens? How to install the package?

Thanks.

_ocaml 5.0.0_

_opam 2.1.4_

_dune 3.6.1_

**Comment by Alizter** [comment, marked as answer]
`dune install` is a command for installing from compiled sources. Are you trying to install it to your opam switch? In which case you probably want to use `opam install base`. If you had a git clone of base, then you could cd to it and dune install.
